staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/service.pb.go

```diff
@@ -0,0 +1,225 @@
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
+
+type DecryptRequest struct {
+	Cipher []byte `protobuf:"bytes,1,opt,name=cipher,proto3" json:"cipher,omitempty"`
+	// Version of the KMS plugin API, now use “v1beta1”
+	Version string `protobuf:"bytes,2,opt,name=version" json:"version,omitempty"`
+}
+
+func (m *DecryptRequest) Reset()                    { *m = DecryptRequest{} }
+func (m *DecryptRequest) String() string            { return proto.CompactTextString(m) }
+func (*DecryptRequest) ProtoMessage()               {}
+func (*DecryptRequest) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{0} }
+
+type DecryptResponse struct {
+	Plain []byte `protobuf:"bytes,1,opt,name=plain,proto3" json:"plain,omitempty"`
+}
+
+func (m *DecryptResponse) Reset()                    { *m = DecryptResponse{} }
+func (m *DecryptResponse) String() string            { return proto.CompactTextString(m) }
+func (*DecryptResponse) ProtoMessage()               {}
+func (*DecryptResponse) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{1} }
+
+type EncryptRequest struct {
+	Plain []byte `protobuf:"bytes,1,opt,name=plain,proto3" json:"plain,omitempty"`
+	// Version of the KMS plugin API, now use “v1beta1”
+	Version string `protobuf:"bytes,2,opt,name=version" json:"version,omitempty"`
+}
+
+func (m *EncryptRequest) Reset()                    { *m = EncryptRequest{} }
+func (m *EncryptRequest) String() string            { return proto.CompactTextString(m) }
+func (*EncryptRequest) ProtoMessage()               {}
+func (*EncryptRequest) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{2} }
+
+type EncryptResponse struct {
+	Cipher []byte `protobuf:"bytes,1,opt,name=cipher,proto3" json:"cipher,omitempty"`
+}
+
+func (m *EncryptResponse) Reset()                    { *m = EncryptResponse{} }
+func (m *EncryptResponse) String() string            { return proto.CompactTextString(m) }
+func (*EncryptResponse) ProtoMessage()               {}
+func (*EncryptResponse) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{3} }
+
+func init() {
+	proto.RegisterType((*DecryptRequest)(nil), "envelope.DecryptRequest")
+	proto.RegisterType((*DecryptResponse)(nil), "envelope.DecryptResponse")
+	proto.RegisterType((*EncryptRequest)(nil), "envelope.EncryptRequest")
+	proto.RegisterType((*EncryptResponse)(nil), "envelope.EncryptResponse")
+}
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ context.Context
+var _ grpc.ClientConn
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the grpc package it is being compiled against.
+const _ = grpc.SupportPackageIsVersion4
+
+// Client API for KmsService service
+
+type KmsServiceClient interface {
```

Comment:
The Go  standard dictates camel case of the form: `KMSServiceClient`.

Reply:
Will update the proto file to fix it.